src/cv_compute/cvScript.sml

```diff
@@ -0,0 +1,254 @@
+(* Support theorems and definitions for the in-kernel compute primitive:
+ * cv_computeLib.cv_compute.
+ *
+ * The kernel compute primitive accepts closed terms written using terms in the
+ * :cv type, using the functions below (e.g., cv_add, cv_if, ...). See the
+ * files in examples/ for examples showcasing usage of the primitive.
+ *
+ *)
+
+open HolKernel boolLib bossLib arithmeticTheory prim_recTheory;
+
+val _ = new_theory "cv";
+
+val op \\ = op THEN;
```

Comment:
Please delete these: they're built-in with these definitions already.

Reply:
Done